crates/modelardb_server/src/common_test.rs

```diff
@@ -0,0 +1,219 @@
+/* Copyright 2023 The ModelarDB Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+//! Implementation of functions, macros, and types used throughout ModelarDB server's tests.
+
+#![cfg(test)]
+
+use std::any::TypeId;
+use std::path::Path;
+use std::sync::Arc;
+
+use datafusion::arrow::array::ArrowPrimitiveType;
+use datafusion::arrow::array::{BinaryArray, Float32Array, UInt64Array, UInt8Array};
+use datafusion::arrow::datatypes::{DataType, Field, Schema};
+use datafusion::arrow::record_batch::RecordBatch;
+use datafusion::execution::context::{SessionConfig, SessionContext, SessionState};
+use datafusion::execution::runtime_env::RuntimeEnv;
+use datafusion::physical_plan::ExecutionPlan;
+use modelardb_common::schemas::COMPRESSED_SCHEMA;
+use modelardb_common::types::{ArrowTimestamp, ArrowValue};
+use modelardb_common::types::{TimestampArray, ValueArray};
+use modelardb_compression::ErrorBound;
+use object_store::local::LocalFileSystem;
+use tokio::sync::RwLock;
+
+use crate::metadata::model_table_metadata::ModelTableMetadata;
+use crate::metadata::MetadataManager;
+use crate::query::ModelTable;
+use crate::storage::{self, StorageEngine};
+use crate::{optimizer, Context, ServerMode};
+
+/// Expected size of the compressed segments produced in the tests.
+pub const COMPRESSED_SEGMENTS_SIZE: usize = 1399;
+
+/// Return a [`Context`] with the metadata manager created by `get_test_metadata_manager()` and
```

Comment:
Change the name of the `test_metadata_manager()` function here.

Reply:
Fixed.